src/main/java/edit_user_public_profile_screens/EditUserPublicProfileResponsePresenter.java

```diff
@@ -0,0 +1,16 @@
+package edit_user_public_profile_screens;
+import edit_user_public_profile_usecase.EditUserPublicProfileOutputBoundary;
+import edit_user_public_profile_usecase.EditUserPublicProfileResponseModel;
+
+
+public class EditUserPublicProfileResponsePresenter implements EditUserPublicProfileOutputBoundary {
+    @Override
+    public EditUserPublicProfileResponseModel prepareSuccessView(EditUserPublicProfileResponseModel EditedChanges) {
+        return EditedChanges;
+    }
+
+    @Override
+    public EditUserPublicProfileResponseModel prepareFailView(EditUserPublicProfileResponseModel FailedChanges) {
```

Comment:
From my understanding the Presenter/Formatter for the prepareFailView should be throwing an error screen. Is there a reason you are returning a response model (FailedChanges)?